maps/adhomai/code/cooking/recipes_pot.dm

```diff
@@ -0,0 +1,211 @@
+/datum/recipe/friedegg
+	appliance = MICROWAVE | POT
```

Comment:
This is not needed, just add it to the original recipe datum.

Reply:
My intent was to make the changes modular, but I can do that.